radio/src/telemetry/frsky_sport.cpp

```diff
@@ -267,6 +267,7 @@ void frskySportSetDefault(int index, uint16_t id, uint8_t instance)
     uint8_t prec = min<uint8_t>(2, sensor->prec);
     telemetrySensor.init(sensor->name, unit, prec);
     if (id == RSSI_ID) {
+      telemetrySensor.filter = 1;
```

Comment:
I remember filter was disabled by default on purpose for smart port telemetry. We receive too few samples in sport and it was introducing too much lag.